### PR TITLE
feat(sms): add example-generated task/servertemplate

### PR DESCRIPTION
**Pull request by valentin-dubreuil**
### Description of your changes

I have:
add example generated 
waiting for the fix of the [creation of an AK/SK pair](https://github.com/FlexibleEngineCloud/terraform-provider-flexibleengine/issues/695) issue to do an example 
- [x] Run `make reviewable test` to ensure this PR is ready for review.

### CODE REVIEW

File: config/sms/config.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+// Configure configures individual resources by adding custom ResourceConfigurators.
+func Configure(p *config.Provider) {
+	// flexibleengine_sms_task
+	// https://registry.terraform.io/providers/FlexibleEngineCloud/flexibleengine/latest/docs/resources/sms_task
+	p.AddResourceConfigurator("flexibleengine_sms_task", func(r *config.Resource) {
+		r.References["vm_template_id"] = config.Reference{
+			Type: "ServerTemplate",
+		}
+
+		p.AddResourceConfigurator("flexibleengine_sms_server_template", func(r *config.Resource) {
```
Comment (azrod) [inline config/sms/config.go:19]: Please lint block

File: config/sms/config.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+			r.References["vpc_id"] = config.Reference{
+				Type: tools.GenerateType("vpc", "Vpc"),
+			}
+			r.References["subnet_ids"] = config.Reference{
+				Type: tools.GenerateType("vpc", "Subnet"),
```
Comment (azrod) [inline config/sms/config.go:24]:
```suggestion
				Type: tools.GenerateType("vpc", "VPCSubnet"),
				Extractor: tools.GenerateExtractor(true, "id"),
```

File: config/sms/config.go [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	// flexibleengine_sms_task
+	// https://registry.terraform.io/providers/FlexibleEngineCloud/flexibleengine/latest/docs/resources/sms_task
+	p.AddResourceConfigurator("flexibleengine_sms_task", func(r *config.Resource) {
+		r.References["vm_template_id"] = config.Reference{
+			Type: "ServerTemplate",
+		}
+
+		p.AddResourceConfigurator("flexibleengine_sms_server_template", func(r *config.Resource) {
+			r.References["vpc_id"] = config.Reference{
+				Type: tools.GenerateType("vpc", "Vpc"),
```
Comment (azrod) [inline config/sms/config.go:21]:
```suggestion
				Type: tools.GenerateType("vpc", "VPC"),
```

**Review by azrod — CHANGES_REQUESTED** [review]

File: examples/sms/task.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+apiVersion: sms.flexibleengine.upbound.io/v1beta1
+kind: Task
+metadata:
+  annotations:
+    meta.upbound.io/example-id: sms/v1beta1/task
+  labels:
+    testing.upbound.io/example-name: migration
```
Comment (azrod) [inline examples/sms/task.yaml:7]: Please respect conventional labelling 

`example_group_resource`

File: examples/sms/task.yaml [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  name: migration
+spec:
+  forProvider:
+    action: start
+    osType: LINUX
+    sourceServerId: ${var.source_server}
```
Comment (azrod) [inline examples/sms/task.yaml:13]: Use random UUID

**Review by azrod — CHANGES_REQUESTED** [review]

**Comment by azrod** [comment]
Please regenerate `list-tested.md`

**Review by dmicheneau — APPROVED** [review]

**Review by azrod — APPROVED** [review]

**Pull request merged**